static/js/components/PersonalTab.js

```diff
@@ -34,9 +38,10 @@ export default class PersonalTab extends React.Component {
     ui:             UIState,
     nextStep:       () => void,
     prevStep:       () => void,
-    programs:       AvailablePrograms,
+    programs:       AvailableProgramsState,
```

Comment:
Why does this need to change? I'm not seeing where we use `programs.getStatus` or anything besides the programs themselves